packages/grafana-flamegraph/src/FlameGraphHeader.tsx

```diff
@@ -221,17 +231,16 @@ const getStyles = (theme: GrafanaTheme2) => ({
     justifyContent: 'space-between',
     width: '100%',
     top: 0,
+    gap: theme.spacing(1),
+    marginTop: theme.spacing(1),
```

Comment:
Instead of each part of the header having padding that then does not look good when collapsed this preserves proper gap whether the elements are in single row or collapsed one under another.